pkg/web/static/js/constants.js

```diff
@@ -246,4 +246,33 @@ const countries = {
   ZW: 'Zimbabwe'
 };
 
-export const countriesArray = Object.entries(countries).map(([value, text]) => ({ text, value }));
\ No newline at end of file
+export const countriesArray = Object.entries(countries).map(([value, text]) => ({ text, value }));
+
+export const networks = {
```

Comment:
The networks object has been moved to make it easier to use in multiple scripts.

Reply:
Good organization!